Description: fix warnings
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2024-03-29
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/nwep.c
+++ b/nwep.c
@@ -62,7 +62,7 @@
 #define oglUniform3f glUniform3f
 #endif
 
-#include "glext.h"
+#include <GL/glext.h>
 
 #include "music/4klang.h"
 #define INTRO_LENGTH (1000ull * MAX_SAMPLES / SAMPLE_RATE)
